frontend/src/components/Collaboration/CodeEditor.js

```diff
@@ -1,4 +1,5 @@
 import React, { useState, useEffect, useRef } from 'react';
+import Spinner from 'react-bootstrap/Spinner';
```

Comment:
Wrong import perhaps? I don't see any usage of Spinner + I think we have a Spinner component within our Component folder.